pkg/plugins/common/kustomize/v1/init.go

```diff
@@ -94,6 +108,30 @@ func (p *initSubcommand) InjectConfig(c config.Config) error {
 	return nil
 }
 
+func (p *initSubcommand) PreScaffold(machinery.Filesystem) error {
+	arch := runtime.GOARCH
+	arch = strings.Replace(arch, "x86_64", "amd64", -1)
```

Comment:
```suggestion
        // It probably will never return x86_64. However, we are here checking the support for the binaries
        // So that, x86_64 means getting the Linux/amd64 binary. Then, we just keep this line to ensure
        // that it complies with the same code implementation that we have in the targets. In case someone
        // call the command inform the GOARCH=x86_64 then, we will properly handle the scenario
        // since it will work successfully and will instal the Linux/amd64 binary via the Makefile target.
	arch = strings.Replace(arch, "x86_64", "amd64", -1)
```